lib/travis/build/script/r.rb

```diff
@@ -0,0 +1,368 @@
+module Travis
+  module Build
+    class Script
+      class R < Script
+        DEFAULTS = {
+          # Basic config options
+          cran: 'http://cran.rstudio.com',
+          warnings_are_errors: false,
```

Comment:
I do wonder if we should make this true by default - I set it to true in all my repos.